common/package.json

```diff
@@ -7,7 +7,9 @@
     "test": "jest"
   },
   "devDependencies": {
+    "@bugsnag/js": "^6.5.2",
```

Comment:
This is already part of the prod dependencies. Does it need to be declared again?

Reply:
it's in the `prod` dependecies of the `client`, it's needed in here because it's now instantiated in `app.js` that is imported by the tests